x-pack/plugins/alerting/common/default_rule_aggregation.ts

```diff
@@ -0,0 +1,183 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import type { AggregationsAggregationContainer } from '@elastic/elasticsearch/lib/api/typesWithBodyKey';
+import {
+  RuleExecutionStatusValues,
+  RuleLastRunOutcomeValues,
+  RuleAggregationFormattedResult,
+} from './rule';
+
+export interface RuleAggregationResult extends Record<string, unknown> {
```

Comment:
As I understood this interface belongs not only to default. Maybe better to transfer it to separate `types` file?

Reply:
I renamed this type to be something more specific. as it only belongs to default.